### ISSUE TITLE
Questions about depreceated commands, PIV and Tutorial for yubikey-agent

### ISSUE THREAD

**Issue by stevek-pro** [jamesog/yubikey-ssh#13]
Thanks for this tutorial. 3 Questions:

- Many if not all commands are depreceated in your Readme.
- What happens if I enter the PIV incorrectly 3 times, will all functions on the Yubikey blocked, including Yubikey OTP Authentication functions?
- You referr to yubikey-agent, but that project does not offer a concise tutorial like you do, any plans on writing one or do you happen to know of one?

Thank you

**Comment by jamesog** [comment]
>     * Many if not all commands are depreceated in your Readme.

Admittedly it's been a while since I've set up a key from scratch, but this surprises me. Which commands did you have trouble with and what makes you say they're deprecated?

>     * What happens if I enter the PIV incorrectly 3 times, will all functions on the Yubikey blocked, including Yubikey OTP Authentication functions?

Please take a look at the official Yubico docs for the answer to this.
https://support.yubico.com/hc/en-us/articles/4402836718866-Understanding-YubiKey-PINs
https://support.yubico.com/hc/en-us/articles/360013779219-Smart-Card-PIN-Unlock-Reset-Operational-Approaches

TL;DR Yubikeys have several "apps" - PIV is one of them, and has its own PIN. My understanding is that OTP is a separate application, but I don't know for certain. I don't use the OTP function personally.

>     * You referr to yubikey-agent, but that project does not offer a concise tutorial like you do, any plans on writing one or do you happen to know of one?

`yubikey-agent` doesn't really need a tutorial because it's designed to completely take over the Yubikey. Per its README, you run `yubikey-agent -setup` and it initialises the key. There's not a lot else to it. The purpose of `yubikey-agent` is to not need a relatively long tutorial like this repo, which is why I generally recommend it instead.

I don't plan on writing a tutorial for it, but you're welcome to ask Filippo by creating an issue in that project.

**Comment by jamesog** [comment]
I'll close this out for now, but feel free to reopen if you can answer my previous question or have something else that doesn't make sense.
